app/src/main/java/app/revanced/integrations/patches/components/LayoutComponentsFilter.java

```diff
@@ -211,7 +212,8 @@ public LayoutComponentsFilter() {
     public boolean isFiltered(@Nullable String identifier, String path, byte[] protobufBufferArray,
                               FilterGroupList matchedList, FilterGroup matchedGroup, int matchedIndex) {
 
-        if (matchedGroup == notifyMe || matchedGroup == inFeedSurvey) return true;
+        if (matchedGroup == notifyMe || matchedGroup == inFeedSurvey || matchedGroup == expandableMetadata) 
```

Comment:
Why is this necessary? Would the last `super.isFiltered` not filter it?

Reply:
Path starts with `home_video_with_context`, which is an exception, so it returns false before it goes to super.isFiltered.